nms/shared/sequelize_models/migrations/20190509000000-make-organization-unique.js

```diff
@@ -27,9 +27,10 @@ module.exports = {
    * @param {QueryInterface} queryInterface
    */
   up: queryInterface => {
-    return queryInterface.addConstraint('Organizations', ['name'], {
+    return queryInterface.addConstraint('Organizations', {
```

Comment:
This was a breaking changed introduced to sequelize 6 and probably the root cause of this issue https://github.com/magma/magma/issues/11291